views/importJS.jade

```diff
@@ -11,3 +11,10 @@ script(src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.11.2/js/bo
 script(src='/bower_components/lodash/lodash.js')
 script(src='/bower_components/jquery.ns-autogrow/dist/jquery.ns-autogrow.min.js')
 script(src='/dist/javascripts/languages.js')
+script.
```

Comment:
do you want to add a conditional for the environment? (i.e. including it only when NODE_ENV=production)